Casks/s/syride-sys-pc-tool.rb

```diff
@@ -0,0 +1,28 @@
+cask "syride-sys-pc-tool" do
+  version "6.33"
+  sha256 "509e6b833c7fc907c3c0989704b941acba6f9d13abe5f0c20834ee4d77eb0c4f"
+
+  url "https://www.syride.com/downloads/SYS_PC_Tool_setup_mac-#{version}.pkg"
+  name "Syride SYS PC Tool"
+  desc "Software for Syride instruments"
+  homepage "https://www.syride.com/"
+
+  livecheck do
+    url "https://www.syride.com/en/software"
+    regex(%r{href="/downloads/SYS[._-]PC[._-]Tool[._-]setup[._-]mac[._-]v?(\d+(?:\.\d+)+)\.pkg}i)
```

Comment:
```suggestion
    url "http://www.syride.com/downloads/SYS_PC_Tool/webconfig-para4.xml"
    regex(/SYS[._-]PC[._-]Tool[._-]setup[._-]mac[._-]v?(\d+(?:[.-]\d+)+)\.pkg/i)
    strategy :xml do |xml, regex|
      url = xml.elements["//sysui_config[@machine='mac']/file"]&.text&.strip
      match = url.match(regex) if url

      match[1]
    end
```

This updates the `livecheck` to use the URL of the in-app updater.

Reply:
Oh, I couldn't find even I use the proxyman, are you using pro version? Then I should consider that.

Reply:
Wow, I learn that I have to learn a few more things! Thought to do a pretty simple addition ;) Thanks for the insights and suggestions! Very helpful and committed as well.

And then I have to find out the previous request how to squash the commits in this PR, I understand what it does, but don't know yet the steps to perform the squash.

Reply:
@IvoElferink 

You can reset and re-commit, push.
```
git reset --mixed master
git add Casks/s/syride-sys-pc-tool.rb
git commit -m "syride-sys-pc-tool 6.33 (new cask)"
git push -f
```

Reply:
Thanks! Done as well

Reply:
@daeho-ro I'm using the free version, but there’s a little trick that @bevanjkay discovered. Sometimes, you can improve your results by starting the app directly from the binary. I'll send you the link to the description on Slack.